wet-boew/geomap/geomap-gc-chckbxrdio-en.html

```diff
@@ -0,0 +1,203 @@
+									</tr>
+									<tr data-geometry="POINT (-75.68937 45.41072)" data-type="wkt">
+										<td>4</td>
+										<td><a href="https://www.wikipedia.org/wiki/Ottawa" title="Ottawa">Ottawa</a></td>
+										<td>883,391</td>
+									</tr>
+									<tr data-geometry="POINT (-113.49590 53.53398)" data-type="wkt">
+										<td>5</td>
+										<td><a href="https://www.wikipedia.org/wiki/Edmonton" title="Edmonton">Edmonton</a></td>
+										<td>812,201</td>
+									</tr>
+									<tr data-geometry="POINT (-79.65 43.60)" data-type="wkt">
+										<td>6</td>
+										<td><a href="https://www.wikipedia.org/wiki/Mississauga" title="Mississauga">Mississauga</a></td>
+										<td>713,443</td>
+									</tr>
+									<tr data-geometry="POINT (-97.14352 49.89375)" data-type="wkt">
+										<td>7</td>
+										<td><a href="https://www.wikipedia.org/wiki/Winnipeg" title="Winnipeg">Winnipeg</a></td>
+										<td>663,617</td>
+									</tr>
+									<tr data-geometry="POINT (-123.10091 49.26428)" data-type="wkt">
+										<td>8</td>
+										<td><a href="https://www.wikipedia.org/wiki/Vancouver" title="Vancouver">Vancouver</a></td>
+										<td>603,502</td>
+									</tr>
+									<tr data-geometry="POINT (-79.76181 43.68686)" data-type="wkt">
+										<td>9</td>
+										<td><a href="https://www.wikipedia.org/wiki/Brampton" title="Brampton">Brampton</a></td>
+										<td>523,911</td>
+									</tr>
+									<tr data-geometry="POINT (-79.86788 43.25717)" data-type="wkt">
+										<td>10</td>
+										<td><a href="https://www.wikipedia.org/wiki/Hamilton,_Ontario" title="Hamilton, Ontario">Hamilton</a></td>
+										<td>519,949</td>
+									</tr>
+								</tbody>
+							</table>
+						</div>
+					</div>
+
+					<div class="panel panel-default">
+						<div class="panel-heading">
+							<div class="panel-title">Bounding Boxes</div>
+						</div>
+						<div class="panel-body">
+							<table id="bbox" aria-label="Bounding Boxes" class="table table-condensed">
+								<caption>Table of bounding box geometries</caption>
+								<thead>
+									<tr>
+										<th>Title</th>
+										<th>Geometry</th>
+									</tr>
+								</thead>
+								<tbody>
+									<tr data-geometry="-136, 61, -118, 70" data-type="bbox">
+										<td>Report on 2011 field activities and collection of ground thermal and active layer data in the Mackenzie Corridor completed under Northwest Territories science licence #14918</td>
+										<td>-136, 61, -118, 70</td>
+									</tr>
+									<tr data-geometry="-92.5, 72.75, -79, 75.5" data-type="bbox">
+										<td>Dinoflagellate cysts from upper cretaceous-lower tertiary sections, Bylot and Devon Islands, Arctic Archipelago</td>
+										<td>-92.5, 72.75, -79, 75.5</td>
+									</tr>
+								</tbody>
+							</table>
+						</div>
+					</div>
+				</div>
+			</section>
+		</div>
+		<!-- Insert Layer Data End -->
+	</div>
+	<!-- GeomapEnd -->
+</section>
```

Comment:
To help our author to implement this large checkbox variation, can you add a little description on how to apply that effect?


```suggestion

<h2>How to implement large checkbox</h2>
<p>Apply the css class <code>large-checkboxes</code> beside the geomap plugin declaration</p>
<pre><code>class="wb-geomap position scaleline geocoder geolocation <strong>large-checkboxes</strong>"
</code></pre>
```